plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -73,9 +73,14 @@ export class EventsProcessor {
             if (!team) {
                 throw new Error(`No team found with ID ${teamId}. Can't ingest event.`)
             }
-
             if (data['event'] === '$snapshot') {
                 if (team.session_recording_opt_in) {
+                    if (this.teamManager.isAboveUsageLimit(team, 'recordings')) {
+                        this.pluginsServer.statsd?.increment('kafka_queue.single_save.snapshot.above_limit', {
```

Comment:
Can we key this somehow by "type" as well (so we see if it is "recordings" or "events")